web/pages/login.tsx

```diff
@@ -22,85 +28,106 @@ export default function LoginScreen() {
-                      <div className="font-opensans text-base">
-                        Username or Email
-                      </div>
-                      <TextInput
-                        formValue={{ ...register("email", { required: true }) }}
-                      />
-                    </div>
-                    <br />
-                    <br />
-                    <div>
-                      <div className="font-opensans text-base">Password</div>
-                      <input
-                        {...register("password", {
-                          required: true,
-                        })}
-                        className="block font-opensans text-base w-80 bg-gray-100 border-1 border-gray-300 p-2"
-                        type="password"
-                      />
+                <div className="flex flex-col w-full sm:w-[55%] sm:order-2">
+                  <div className={`${emailError ? "mb-[1.2%] sm:mb-[1.6%]" : "mb-[7%]"}`}>
+                    <p className="mb-[2%] sm:mb-[1%]">
+                      Email
+                    </p>
+                    <TextInput errorMsg={emailError} onChange={(event) => {
+                      setEmail(event);
+                      setEmailError("")
+                    }}></TextInput>
+                  </div>
+                  <div className={`${passwordError ? "mb-[1.2%] sm:mb-[1.6%]" : "mb-[2%]"}`}>
+                    <p className="mb-[2%] sm:mb-[1%]">
+                      Password
+                    </p>
+                    <TextInput inputType="password" errorMsg={passwordError} onChange={(event) => {
+                      setPassword(event);
+                      setPasswordError("")
+                    }}></TextInput>
+                  </div>
+                  <div className="flex justify-end mb-[12%] sm:mb-[8%]">
+                    <div className="w-auto text-center text-mbb-pink text-sm font-semibold font-opensans">
+                      Forgot Password
                     </div>
-                    <br />
-                    <Button text="Log In" submit={true} />
-                    <Button
-                      icon={<AiFillGoogleCircle />}
-                      text="Log In with Google"
-                      onClick={loginWithGoogle}
-                    />
                   </div>
-                </form>
+                  <div className="mb-[10%] sm:mb-[5%]">
+                    <Button text="Log In" onClick={() => {
+                      if (email && password) {
+                        loginWithCredentials(email, password).then((e) => {
+                        if (e.success) {
+                          //route to next page
+                        } else {
+                          //error
+                        }
+                      })
+                      } else {
+                        if (!email) {
+                          setEmailError("Please enter a email")
+                          setErrorBanner("true")
```

Comment:
The error banner only needs to be shown for overall errors like if their login info is incorrect or if there's a server error, so for input-specific errors (blank email or password), use only the error text